x-pack/plugins/maps/common/map_saved_object_type.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { SavedObject } from '../../../../src/core/server';
+
+export type MapSavedObjectAttributes = {
```

Comment:
Why are we changing from `interface` to `type`? Similar refs in the code use an interface: https://github.com/elastic/kibana/blob/f1272b5ffe34d7efacb359260e930b872e8e4b06/src/core/server/saved_objects/types.ts

Reply:
We have started the precedent in https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/maps/common/descriptor_types.d.ts to use `type` when defining something with just properties and `interface` when defining an actual interface that contains methods that need to be implemented.